python/tvm/driver/tvmc/common.py

```diff
@@ -17,6 +17,11 @@
 """
 Common utility functions shared by TVMC modules.
 """
+import logging
```

Comment:
Use the standard logging, such as `logger = logging.getLogger("TVMC")`.

Reply:
Yep. I also added a note to migrate the compiler module to the common logger, in a separate PR.